Automattic/wp-calypso
# Revert "Revert "Add instructional 'Fonts' section to customizer when fonts are only modifiable from 'Global Styles' (#54191)" (#54658)"

This reintroduces pull request [#54191](https://github.com/Automattic/wp-calypso/pull/54191)

This was originally reverted in order to fix linting errors.

### Testing instructions
#### Simple Site Testing Instructions

1. Host a calypso app locally: `cd wp-calypso; yarn && yarn start`
2. Host the editing toolkit plugin as well. `cd wp-calypso/apps/editing-toolkit; yarn dev --sync`
3. Make sure your hosts file is pointing wordpress.com and public-api.wordpress.com to your sandbox, as well as the specific site you're about to test with.
4. Make sure you have an active SSH connection to your sandbox
4. Perform all the following steps using your locally hosted calypso app.
3. Activate the theme: twenty-twenty one
4. Go to customizer
5. Expand the 'Fonts' section
7. Click the link which says 'Click here to open the Block Editor and change your fonts.'
8. It should open your homepage and expand the global styles section

#### Atomic Testing Instructions

1. Create an atomic site
2. Make sure your hosts file is pointing to production servers
3. Download the zipped version of the editing toolkit plugin from the build for this pull request
4. Install/activate this plugin (You may need to deactivate the existing editing toolkit plugin first)
3. Activate twenty twenty-one
4. Go to customizer
5. Expand the 'Fonts' section
7. Click the link which says 'Click here to open the Block Editor and change your fonts.'
8. It should open your homepage in the block editor (the global styles sidebar may or may not be open)
1. `?openSidebar=globalStyles` should be present in your current URL
